app/src/main/java/edu/temple/templetag/fragments/MapFragment.java

```diff
@@ -37,41 +37,56 @@
 import edu.temple.templetag.HomeActivity;
 import edu.temple.templetag.R;
 import edu.temple.templetag.Tag;
+import edu.temple.templetag.tools.DistanceCalculator;
 
 public class MapFragment extends Fragment implements OnMapReadyCallback {
     private static final String ARG_PARAM1 = "tags";
```

Comment:
Could we update the names for these constant keys from ARG_PARAM# to be like TAGS and TAG so things are more readable below?